src/Error/ExceptionRenderer.php

```diff
@@ -184,7 +184,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
On a note, I just wanted to add info about the new behavior of stack traces only going to debug.log/error.log when using XmlView/JsonView to have a way for the developer to have good exception reporting...
... and double checked the logs and don't see anything. If debug is on, http://devicesdbservices.dev/api/search/manufacturers will show Unauthorized while http://devicesdbservices.dev/api/search/manufacturers.xml and http://devicesdbservices.dev/api/search/manufacturers.json won't write anything to debug.log.
Shouldn't these requests do that?